cmd/bootstrap/cmd/seal.go

```diff
@@ -40,7 +40,7 @@ func constructRootResultAndSeal(
 		DKGPhase3FinalView: firstView + flagNumViewsInStakingAuction + flagNumViewsInDKGPhase*3 - 1,
 		Participants:       participants.Sort(order.Canonical),
 		Assignments:        assignments,
-		RandomSource:       getRandomSource(flagBootstrapRandomSeed),
```

Comment:
`getRandomSource` validates the seed length, why do we remove the validation?

Reply:
`getRandomSource` just returns a copy of the input seed with a different length: crops it if it is long, and padds with zeros if it's short. 
The padding part is dangerous: it makes the seed look long enough, but doesn't use a proper entropy for it. I prefer to use the originally generated seed and not "correct" it.